aws-logs-querydefinitions/README.md

```diff
@@ -0,0 +1,12 @@
+# AWS::Logs::QueryDefinitions
```

Comment:
Should we have some content in here describing how to build/test and anything else related to the general developer workflow? Perhaps that content belongs on the main README and maybe out of scope here? It does seem like the documentation in this repo could use some TLC.

Reply:
yes agreed, good call out. I'll add in next PR